test/broken-pipe.test.js

```diff
@@ -1,47 +1,37 @@
 'use strict'
 
-const t = require('tap')
+const test = require('node:test')
+const assert = require('node:assert')
 const { join } = require('node:path')
 const { fork } = require('node:child_process')
+const tspl = require('@matteo.collina/tspl')
 const { once } = require('./helper')
 const pino = require('..')
 
 if (process.platform === 'win32') {
-  t.skip('skipping on windows')
+  console.log('skipping on windows')
   process.exit(0)
 }
 
-if (process.env.CITGM) {
-  // This looks like a some form of limitations of the CITGM test runner
-  // or the HW/SW we run it on. This file can hang on Node.js v18.x.
-  // The failure does not reproduce locally or on our CI.
-  // Skipping it is the only way to keep pino in CITGM.
-  // https://github.com/nodejs/citgm/pull/1002#issuecomment-1751942988
-  t.skip('Skipping on Node.js core CITGM because it hangs on v18.x')
-  process.exit(0)
-}
```

Comment:
I would keep skipping this on CITGM

Reply:
But v18 isn't supported?

Reply:
I'm a bit unsure this test would actually run on the servers CITGM runs on, so it's better to keep skipping it.